magento/inventory
# MSI-2170 Reservation CLI / Improve compensation create command

### Description (*)
Improves CLI for compensation creation in order pass input from `list-inconsistencies` to the `create-compensations` command.

### Fixed Issues (if relevant)
1. magento-engcom/msi#2170: Reservations CLI: Compensation

### Manual testing scenarios (*)
1. Create some orders in complete and incomplete state
2. Remove or manipulate reservations in DB
3. Run `bin/magento inventory:reservation:list-inconsistencies` to see inconsistencies
4. Run `bin/magento inventory:reservation:list-inconsistencies -r | bin/magento inventory:reservation:create-compensations` in order to pass inconsistencies to compensation command.
5. Run `bin/magento inventory:reservation:list-inconsistencies` again - no inconsistencies should be present

### Contribution checklist (*)
